insights/src/main/kotlin/com/algolia/instantsearch/insights/webservice/WebService.kt

```diff
@@ -10,5 +10,6 @@ internal interface WebService {
         val code: Int
     )
 
-    fun sendEvent(event: Event): Response
+    fun send(events: List<Event>): Response
+    fun send(event: Event) = send(listOf(event))
```

Comment:
Looks like a nice case for a `vararg` function instead of these two ?

Reply:
Yes, but failed to. Feel free to give it a try, or let's discuss on monday :)